intrinsicsolver/fixFunctions.go

```diff
@@ -13,12 +13,16 @@ import (
 
 /*
 FixFunctions : takes []byte file and firstly converts all single quotation marks to double ones (anything between single ones is treated as the rune in GoLang),
```

Comment:
Do we need `FixFunctions :` at the beginning? I thought we don't.

Reply:
The linter is advising that *exported* function description must be in `// Funtion name : description` manner.